src/Capabilities_Command.php

```diff
@@ -66,38 +64,35 @@ class Capabilities_Command extends WP_CLI_Command {
 	 */
 	public function list_( $args, $assoc_args ) {
 		$role_obj = self::get_role( $args[0] );
-		
+
 		$show_grant = ! empty( $assoc_args['show-grant'] );
 
 		if ( $show_grant ) {
 			array_push( $this->fields, 'grant' );
 			$capabilities = $role_obj->capabilities;
-		} 
-		else {
+		} else {
 			$capabilities = array_filter( $role_obj->capabilities );
 		}
-		
+
 		$output_caps = array();
 		foreach ( $capabilities as $cap => $grant ) {
-			$output_cap = new StdClass;
+			$output_cap = new StdClass();
 
-			$output_cap->name = $cap;
+			$output_cap->name  = $cap;
 			$output_cap->grant = ( $grant ) ? 'true' : 'false';
-			
+
 			$output_caps[] = $output_cap;
 		}
 
 		if ( 'list' === $assoc_args['format'] ) {
 			foreach ( $output_caps as $cap ) {
 				if ( $show_grant ) {
 					WP_CLI::line( implode( ',', array( $cap->name, $cap->grant ) ) );
-				} 
-				else {
+				} else {
 					WP_CLI::line( $cap->name );
 				}
 			}
-		}
-		else {
+		} else {
 			$formatter = new \WP_CLI\Formatter( $assoc_args, $this->fields );
```

Comment:
Please import the dependencies at the top and use relative classes/namespaces.
```suggestion
			$formatter = new Formatter( $assoc_args, $this->fields );
```